lambdanote/src/Components/CreateForm.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react';
+import { Button, Form, FormGroup, Input } from 'reactstrap';
+
+export default class CreateForm extends Component {
+    constructor() {
+        super();
+        this.state = {
+            notes: [],
+            title: '',
+            text: ''
+        }    
+    }
+
+    handdleInputChange = e => {
+        this.setState({ [e.target.name]: e.target.value})
+    }
+
+    createNewInput = (event)=> {
+        event.preventDefault();
+        const notes = this.state.notes;
+        // const newNotes = { 
+        //     title: this.state.title, 
+        //     text: this.state.text}
+        // notes.push(newNotes);
+        this.setState({ notes, title: '', text: ''})
```

Comment:
Nice work! Just remember you'll have to change this up when you start storing data in a server (or persisting it from session to session in the browser). It will also need to be changed up if/when you add Redux to the mix. Something to keep in mind :grinning: